etl/steps/data/garden/artificial_intelligence/2023-06-21/epoch.meta.yml

```diff
@@ -0,0 +1,89 @@
+dataset:
+  title: Parameter, Compute and Data Trends in Machine Learning (Epoch, 2023)
+  description: &desc |
+    We update this chart with the latest available data from our source every month.
+
+    The authors selected the AI systems for inclusion based on the following necessary criteria:
+    — Have an explicit learning component
+    — Showcase experimental results
+    — Advance the state of the art
+
+    In addition, the systems had to meet at least one of the following notability criteria:
+    — Paper has more than 1000 citations
+    — Historical importance
+    — Important state-of-the-art advance
+    — Deployed in a notable context
+
+    The authors note that: "For new models (from 2020 onward) it is harder to assess these criteria, so we fall back to a subjective selection. We refer to models meeting our selection criteria as 'milestone models.'"
+  licenses:
+  - name: Public domain
+  sources:
+  - name: Epoch (2023)
+    description: *desc
+    url: https://epochai.org/mlinputs/visualization
+    date_accessed: '2023-06-21'
+    publication_date: '2023-06-06'
+    publication_year: 2023
+    published_by: 'Epoch, Parameter, Compute and Data Trends in Machine Learning.
+      Published online at epochai.org. Retrieved from: https://epochai.org/mlinputs/visualization'
```

Comment:
This was already defined in the snapshot step, right?

If so, you could remove this and propagate it in the code. It is propagated when you create the Garden dataset with `create_dataset` and use the `default_metadata` field (which I see that you set to `None`).